http/src/client/mod.rs

```diff
@@ -424,12 +424,22 @@ impl Client {
         UpdateGuildChannelPositions::new(self, guild_id, channel_positions)
     }
 
-    pub fn guild_embed(&self, guild_id: GuildId) -> GetGuildEmbed<'_> {
-        GetGuildEmbed::new(self, guild_id)
+    #[deprecated(note = "Please use the guild_widget method instead.")]
+    pub fn guild_embed(&self, guild_id: GuildId) -> GetGuildWidget<'_> {
+        GetGuildWidget::new(self, guild_id)
     }
 
-    pub fn update_guild_embed(&self, guild_id: GuildId) -> UpdateGuildEmbed<'_> {
-        UpdateGuildEmbed::new(self, guild_id)
+    pub fn guild_widget(&self, guild_id: GuildId) -> GetGuildWidget<'_> {
+        GetGuildWidget::new(self, guild_id)
+    }
+
+    #[deprecated(note = "Please use the update_guild_widget method instead.")]
```

Comment:
Think it's worth it to have deprecation notices? We're pre-0.0.1 and there's not really too many people to notify about it, so I think we should just not bother or it'll stay around for a while.

Reply:
Does discord still support these? If so I can see a reason for leaving them, but adding a deprecation notice.

However I'd be in favor of removing them as Vivian said above, I don't think we need to keep around old stuff while we remain in pre-0.0.1.

Reply:
I agree with not needing deprecation notices while Twilight is still pre-0.0.1.